cpp/ql/src/semmle/code/cpp/models/implementations/StdPair.qll

```diff
@@ -0,0 +1,60 @@
+/**
+ * Provides models for the C++ `std::pair` class.
+ */
+
+import semmle.code.cpp.models.interfaces.Taint
+
+/**
+ * Additional model for `std::pair` constructors.
+ */
+class StdPairConstructor extends Constructor, TaintFunction {
+  StdPairConstructor() { this.getDeclaringType().hasQualifiedName("std", "pair") }
```

Comment:
Yes, I intend to look into modelling classes as well as methods (with a view to better performance and perhaps cleaner code), however I'm reluctant to do this before I've finished the models for `map`, `set` and so on as it will lead to merge conflicts.

For now I've accepted your change.